utility/src/main/java/org/oppia/util/logging/firebase/FirebaseEventLogger.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.util.logging.firebase
+
+import android.content.Context
+import android.os.Bundle
+import com.google.firebase.analytics.FirebaseAnalytics
+import org.oppia.app.model.EventLog
+import org.oppia.util.logging.EventBundleCreator
+import org.oppia.util.logging.EventLogger
+import javax.inject.Singleton
+
+/** Logger for event logging to Firebase Analytics. */
+@Singleton
+class FirebaseEventLogger(
+  private val firebaseAnalytics: FirebaseAnalytics
+) : EventLogger {
+  private var bundle = Bundle()
+
+  /** Logs an event to Firebase Analytics. */
+  override fun logEvent(context: Context, eventLog: EventLog) {
+    bundle = EventBundleCreator().createEventBundle(eventLog)
```

Comment:
Inject EventBundleCreator like how you have done for firebaseAnalytics instead

Reply:
Done.

Reply:
This is not done yet?

Reply:
No, it's done.

Reply:
Done.